client/src/app/site/pages/meetings/view-models/view-meeting.ts

```diff
@@ -139,6 +139,16 @@ export class ViewMeeting extends BaseHasMeetingUsersViewModel<Meeting> {
     public canBeEnteredBy(user: ViewUser): boolean {
         return !this.locked_from_inside || user.group_ids(this.id).length > 0;
     }
+
+    public getStatus(): string[] {
+        const status: string[] = [];
+        this.isArchived ? status.push(`isArchived`) : status.push(`isNotArchived`);
+        this[`enable_anonymous`] ? status.push(`isAnonymous`) : status.push(`isNotAnonymous`);
+        if (this[`locked_from_inside`]) {
+            status.push(`IsClosed`);
```

Comment:
```suggestion
        if (this[`locked_from_inside`]) {
            status.push(`isLockendFromInside`);
```

Do not introduce new naming.